app/models/user.py

```diff
@@ -8,8 +8,8 @@
 
 class Permission:
     GENERAL = 0x01
```

Comment:
I don't think so. It's here in case there's ever another user role in the future. It just helps describe permissions shared by Vendors and Merchants.

We probably COULD get rid of it, but testing for the `GENERAL` permission is better practice than testing for `is_logged_in`. They both mean the same thing right now, but this lets us make the system more flexible in the future without major refactoring.